lightning/impl/base.py

```diff
@@ -52,7 +59,7 @@ def predict_proba(self, X):
             prob += 1
             prob /= 2
         else:
-            raise NotImplementedError("predict_(log_)proba only supported when"
+            raise NotImplementedError("predict_proba only supported when"
```

Comment:
Because code flow should never get to this exception - AttributeError is raised earlier. If for some reason _predict_proba is used instead of predict_proba then with unsupported loss function an exception is still raised (`prob` variable is not defined); it is harder to interpret that NotImplementedError though. So maybe keeping it is fine :)